config-example.yml

```diff
@@ -79,6 +79,26 @@ introspection:
   #   - path/to/schema/part2.gql
   schemaFile: path/to/schema.gql
 
+
+  # Some helpful options for those who are an "SDL-first" shop and/or want to get your metadata into
+  # SpectaQL during the SDL ingestion.
+  spectaqlDirective:
+    # Boolean indicating whether to enable and process the @spectaql directive
+    #
+    # Default: true
+    enable: true
```

Comment:
If it's enabled by default, all good if it's not there? Like this wont break existing installs out there, will it?

Reply:
Yeah it's pretty flexible...